src/app/components/Timer.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component, PropTypes } from 'react';
+
+export default class Timer extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            secondsElapsed: props.from
+        };
+    }
+
+    tick() {
+        this.setState({
+            secondsElapsed: this.state.secondsElapsed + 1
+        });
+    }
+
+    componentDidMount() {
+        this.interval = setInterval(() => this.tick(), 1000);
+    }
+
+    componentWillUnmount() {
+        clearInterval(this.interval);
+    }
+
+    render() {
+        return (
+            <div>Seconds Elapsed: {this.state.secondsElapsed}</div>
+        );
+    }
+}
+
+Timer.propTypes = {
```

Comment:
So choosing `latest` means that we’re not going to use class props, right?

To be clear, I’m not that much into `static whatever = {}` but I really like that it can be written in a prominent place on the very top of the class definition.

But considering other use cases, it’s not likely that we’ll use classes outside of React and even in React we’ll use stateless functional components more and more. So you’re propably right, we can live without static props.